packages/bits/demo/src/components/demo/paginator/paginator-test/paginator-test.component.html

```diff
@@ -0,0 +1,10 @@
+
+<div class="container">
```

Comment:
Can all of the paginators on this page go inside the container div?

Reply:
Sure.